heal-commands.go

```diff
@@ -309,15 +309,16 @@ type MRFStatus struct {
 
 // BgHealState represents the status of the background heal
 type BgHealState struct {
+	// Total items scanned by the continous background healing
 	ScannedItemsCount int64
-
+	// Disks currently in heal states
 	HealDisks []string
-
 	// SetStatus contains information for each set.
 	Sets []SetStatus `json:"sets"`
-
 	// Endpoint -> MRF Status
 	MRF map[string]MRFStatus `json:"mrf"`
+	// Parity per storage class
+	SCParity map[string]int `json:"scParity"`
```

Comment:
```suggestion
	SCParity map[string]int `json:"sc_parity"`
```
Snake case is used elsewhere and JSON default.